docs/guides/building/arm-cross-compile.md

```diff
@@ -0,0 +1,162 @@
+          commands: |
+            # Rust complains (rightly) that $HOME doesn't match eid home
+            export HOME=/root
+            # Workaround to CI worker being stuck on Updating crates.io index
+            export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
+            # Install setup prerequisites
+            apt-get update -y --allow-releaseinfo-change
+            apt-get upgrade -y
+            apt-get autoremove -y
+            apt-get install curl
+            curl https://sh.rustup.rs -sSf | sh -s -- -y
+            . "$HOME/.cargo/env"
+            curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash
+            # Install framework specific packages
+            apt-get install -y nodejs
+            npm install next@latest react@latest react-dom@latest eslint-config-next@latest
+            # Install build tools and tauri-cli requirements
+            apt-get install -y libwebkit2gtk-4.0-dev build-essential wget libssl-dev libgtk-3-dev libayatana-appindicator3-dev librsvg2-dev
+            cargo install tauri-cli
+            # Install frontend dependencies
+            npm install
+            # Build the application
+            cargo tauri build
+      - name: Upload deb bundle
+        uses: actions/upload-artifact@v3
+        with:
+          name: Debian Bundle
+          path: ${{ github.workspace }}/target/release/bundle/deb/tauri_1.4_arm64.deb
+```
+
+Adjust the `path` variable to match your application's version and name:
+`${{ github.workspace }}/target/release/bundle/deb/[name]_[version]_arm64.deb`.
+
+### Final Notes
+
+During the cross-compilation process, Rust might attempt to run your app at the end, but it will fail since the build is targeted for ARM architecture, and you are likely on a different platform. While this error is expected, it's important to note that if Rust encounters an error during the execution, the entire GitHub Action will also fail, resulting in no executable being generated.
+
+To avoid this issue and ensure a smooth cross-compilation process, you can take the following steps:
+
+1. Configure `tauri.conf.json`: Inside the `src-tauri` folder, you can customize the `tauri.conf.json` file to specify the target platforms. Remove `"appimage"` from the `"targets"` array and keep only the desired targets for your ARM-based device. For example:
+   ```json
+   "targets": ["deb", "nsis", "msi", "app", "dmg", "updater"],
+   ```
+
+By adjusting the `tauri.conf.json` file, you instruct Tauri to exclude the "appimage" target, which is irrelevant for ARM devices. This way, Rust won't attempt to execute the non-ARM-compatible "appimage" target during the cross-compilation process, preventing any failures and ensuring the successful creation of the `.deb` package for your ARM-based device.
+
+With these modifications, you can confidently use the GitHub Action to automatically build an ARM-compatible `.deb` package for your Tauri application. Happy coding!
```

Comment:
I will copy my comment from discord:

> Also, the Final Notes section: The fix itself is correct, but the problem you describe isn't. It's not trying to run the app at the end or something. This is solely the appimage bundler not supporting arm targets resulting in the download of the necessary bundler tools failing
and nothing of it is related to rust, that part is already done at that point
And lastly on that topic, i think it makes more sense to completely remove that section and add a warning ala "appimages currently unsupported, disable it via" at the top similar to the ubuntu2004 warning

Reply:
I removed the entire section as requested and added a warning at the top.